content/applications/finance/documents.rst

```diff
@@ -0,0 +1,127 @@
+:show-content:
+
+=========
+Documents
+=========
+
+**Odoo Documents** allows you to store, view and manage files within Odoo.
+
+You can upload any type of file (max 64MB per file on Odoo Online/SaaS), and organize them in
+folders known as workspaces.
+
+.. seealso::
+   - `Odoo Documents: product page <https://www.odoo.com/app/documents>`_
+
+Workflow actions
+================
+
+Workflow actions are automated actions you can create and customize at the level of each
+workspace. They appear next to a file whenever it meets the criteria you set. That way you can,
+for example, add tags to a file or move it to another workspace with a single click. Workflow
+actions help you streamline the management of your documents and your overall business operations.
+
+Create workflow actions
+-----------------------
+
+To create workflow actions, go to :menuselection:`Documents --> Configuration --> Workspaces` and
+select the workspace where the action should apply. Click on the *Actions* smart button, and then
+on *Create*.
+
+.. image:: documents/access-workflow-actions.png
+   :align: center
+   :alt: Workflow actions smart button in Odoo Documents
+
+.. note::
+   An action applies to all *Child Workspaces* under the *Parent Workspace* you selected (i.e., to
+   all subfolders under the selected folder).
+
+.. tip::
+   If you use the :doc:`developer mode <../general/developer_mode/activate>`, you can directly
+   access all your actions by going to :menuselection:`Documents --> Configuration --> Actions`.
+
+Set the conditions
+------------------
+
+After naming your workflow action, you can set the conditions that trigger the appearance of the
+action button on the right-side panel when selecting a file.
+
+There are three basic types of conditions you can set:
+
+#. **Tags**: you can both use the *Contains* and *Does not contain* conditions, meaning the files
+   *must have* or *mustn't have* the tags set here.
+
+#. **Contact**: the files must be associated with the contact set here.
+
+#. **Owner**: the files must be associated with the owner set here.
+
+.. image:: documents/basic-condition-example.png
+   :align: center
+   :alt: Example of a workflow action's basic condition in Odoo Documents
+
+.. tip::
+   If you don't set any conditions, the action button appears for all files located inside the
+   selected workspace.
+
+Advanced condition type: domain
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. important::
+   It is recommended to have some knowledge of Odoo development to properly configure *Domain*
+   filters.
```

Comment:
I added this as a suggestion in the previous review but I guess you didn't see it:
put "properly" at the end of the sentence.
```suggestion
   It is recommended to have some knowledge of Odoo development to configure *Domain* filters
   properly.
```

Reply:
All good then :D 
thanks